pkgs/os-specific/linux/kernel/linux-lqx.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, buildLinux, ... } @ args:
+
+let
+  version = "5.9.16";
+in
+
+buildLinux (args // {
+  modDirVersion = "${version}-lqx1";
+  inherit version;
+  isZen = true;
+
+  src = fetchFromGitHub {
+    owner = "zen-kernel";
+    repo = "zen-kernel";
+    rev = "v${version}-lqx1";
+    sha256 = "0ljvqf91nxpql98z75bicg5y3nzkm41rq5b0rm1kcnsk0ji829ps";
+  };
+
+  extraMeta = {
+    branch = "5.9/master";
+    maintainers = with stdenv.lib.maintainers; [ atemu ];
```

Comment:
Please add a description. What is "linux-lqx"? Why would I want to use it? How does it differ from the vanilla linux, how does it differ from linux-zen?